File: SDKTest/SDKTest/ViewController.m
//
//  ViewController.m
//  SDKTest
//
//  Created by gemvary_mini_2 on 2018/11/19.
//  Copyright © 2018 happylong. All rights reserved.
//

#import "ViewController.h"
#import <MLSDK/MLSDK.h>

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}



- (IBAction)clickedAction:(UIButton *)sender {
    
    UIViewController *buttonVC = [MLAction creatFirstVC];
    [self.navigationController pushViewController:buttonVC animated:YES];
    
}



@end

File: FilesClippers/FilesClippers/ViewController.swift
//
//  ViewController.swift
//  FilesClippers
//
//  Created by SongMenglong on 2018/12/15.
//  Copyright © 2018 gemvary. All rights reserved.
//

import UIKit
import LLBlueTooth

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
        // 初始化蓝牙管理者
        LLBlueTooth.shared.initCentralManager()
        
        LLBlueTooth.shared.scanForPeripheralsWithTime(time: 3)
        
    }
    
}

